master/buildbot/test/unit/test_process_properties.py

```diff
@@ -1366,53 +1366,56 @@ def setUp(self):
         self.props = Properties()
         self.build = FakeBuild(props=self.props)
 
+    @defer.inlineCallbacks
     def test_renderer(self):
         self.props.setProperty("x", "X", "test")
-        d = self.build.render(
-            renderer(lambda p: 'x%sx' % p.getProperty('x')))
-        d.addCallback(self.assertEqual, 'xXx')
-        return d
 
-    def test_renderer_called(self):
```

Comment:
Why did you remove this test? looks like an overlook.

Reply:
Yeah, the first proposal that I had in mind was making the renderable callable instead of adding the `withArgs` method. I've removed the test and forgot the add it after figuring out that the callable API would be completely unreadable and unexpected. Thanks for noticing this! Fixed now.